opencti-platform/opencti-front/lang/front/en.json

```
@@ -44,6 +44,7 @@
   "Activate access restriction": "Activate access restriction",
   "Activate/Deactivate default values": "Activate/Deactivate default values",
   "Activated": "Activated",
+  "Activating this inference rule will automatically generate new relationships on your existing and future data. Please be aware that enabling this rule may impact your platform's data and performance.": "Activating this inference rule will automatically generate new relationships on your existing and future data. Please be aware that enabling this rule may impact your platform's data and performance.",
```

```suggestion
  "Activating this rule will automatically generate new relationships or new data based on your existing and future data. Please be aware that enabling this rule may impact your platform's data and performance.": "Activating this rule will automatically generate new relationships or new data based on your existing and future data. Please be aware that enabling this rule may impact your platform's data and performance.",
```

It's message for all rules, and all rule are nt inference one so I think it shoud be a bit more generic.

---

done